test/test_helper.exs

```diff
@@ -4,8 +4,7 @@ ExUnit.configure exclude: [integration: true, consumer_group: true, server_0_p_9
 
 defmodule TestHelper do
   def generate_random_string(string_length \\ 20) do
-    :random.seed(:os.timestamp)
-    Enum.map(1..string_length, fn _ -> (:random.uniform * 25 + 65) |> round end) |> to_string
+    Enum.map(1..string_length, fn _ -> (:rand.uniform * 25 + 65) |> round end) |> to_string
```

Comment:
We use OTP 19 + elixir 1.4 everywhere right now. I can certainly put :random.uniform back if we want to keep supporting OTP 17